test/Core/Eval/Base.hs

```diff
@@ -28,7 +28,7 @@ coreEvalAssertion mainFile expectedFile trans testTrans step = do
   case r of
     Left err -> assertFailure (show (pretty err))
     Right (_, Nothing) -> do
-      step "Compare expected and actual program output"
+      step "Compare expected an actual program output"
```

Comment:
Typo

Reply:
I must have messed something going back and forth between my two laptops... I already fixed some of this issues. Sorry about that, I'll double check everything again